android/src/main/java/com/reactnativehmssdk/HMSManager.kt

```diff
@@ -578,15 +818,33 @@ class HMSManager(reactContext: ReactApplicationContext) :
         .emit(event, data)
   }
 
+  override fun onHostResume() {
+    Log.i("HMSManager", "Host Resumed")
+  }
+
+  override fun onHostPause() {
+    Log.i("HMSManager", "Host Paused")
+  }
+
+  override fun onHostDestroy() {
+    Log.i("HMSManager", "Host Destroyed")
+  }
+
   override fun onActivityCreated(activity: Activity, savedInstanceState: Bundle?) {}
 
   override fun onActivityStarted(activity: Activity) {}
 
-  override fun onActivityResumed(activity: Activity) {}
+  override fun onActivityResumed(activity: Activity) {
+    Log.i("HMSManager", "Activity Resumed")
```

Comment:
logs to be removed

Reply:
done in commit - f105fe3d91ad50cf3916a645b8e93b524b8c33c1